filenames.gni

```diff
@@ -669,6 +669,8 @@ filenames = {
     "shell/common/language_util.h",
     "shell/common/logging.cc",
     "shell/common/logging.h",
+    "shell/common/mac_util.h",
+    "shell/common/mac_util.mm",
```

Comment:
These should be added to lib_sources_mac: https://github.com/electron/electron/blob/afa1fdb2576551212d44609f681615e465878faa/filenames.gni#L122

Reply:
@jkleinsc moved in cfc63651697ed25d18efd1d278026a10a48dc398